local.go

```diff
@@ -0,0 +1,549 @@
+package imgutil
+
+import (
+	"archive/tar"
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/pkg/errors"
+)
+
+type LocalImage struct {
+	RepoName         string
+	Docker           *client.Client
+	Inspect          types.ImageInspect
+	layerPaths       []string
+	Stdout           io.Writer
```

Comment:
It looks like Stdout isn't used, now that logging has been removed from the image

Reply:
Done.